### ISSUE TITLE
Chore/Feat: Reset & Document `@home` Wifi Network

### ISSUE THREAD

**Issue by nelsonic** [dwyl/home#229]

**Comment by nelsonic** [comment]
UPS installed and working well: https://github.com/dwyl/home/issues/167 🔋 ✅ 

Going to continue trying to get the `@home` network working now... 🧑‍💻 ⏳

**Comment by nelsonic** [comment]
`@home` network wasn't working because the `USG` (firewall) failed to adopt:
<img width="1654" alt="home-network-USG-adoption-failed" src="https://user-images.githubusercontent.com/194400/234842045-38530412-d538-44d3-96d5-764b7be3fe9f.png">

There is a firmware update so I'm applying it:
<img width="1818" alt="home-network-usg-firmware-updating" src="https://user-images.githubusercontent.com/194400/234842279-07c2e8f3-782d-4da5-92cc-c31236dcde81.png">

Don't worry, all these IP Addresses will be changed before the network setup is done.

**Comment by nelsonic** [comment]
Not much I can do while this is updating ...

<img width="1818" alt="image" src="https://user-images.githubusercontent.com/194400/234842911-306329a7-6944-4a60-a182-c147e124e8ba.png">

Heading to lunch. 🥗

**Comment by nelsonic** [comment]
Adoption failed (again) ...

<img width="1792" alt="adoption-failed-all" src="https://user-images.githubusercontent.com/194400/234875459-9daf0e75-b433-4873-a9d9-e9fb722850ad.png">

Reading: https://community.ui.com/questions/SOLVED-USG-3P-adoption-failed/dd87c48a-eade-4f25-9141-6b8a73ee7d4e

**Comment by nelsonic** [comment]
Nothing I read on the UI forums was minimally useful. So I did a manual reset. 💥 
But nothing in the docs warned me that I had to "forget" each device before doing each reset ... 🙄 
Anyway, still not able to connect to the internet. ⏳

**Comment by nelsonic** [comment]
The good news is: after resetting the MEO router which cleared all the historical clients the speed while connected to WiFI is much faster again:

<img width="979" alt="image" src="https://user-images.githubusercontent.com/194400/235010149-ba24383a-76c7-4cf4-a9f4-3ee84e537649.png">

**Comment by nelsonic** [comment]
Good news the `@home` network is now operational:

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/194400/235011423-3fe5208a-8fa0-4aa8-8f19-2cd5f92f6e35.png">

Decent speed:

<img width="816" alt="image" src="https://user-images.githubusercontent.com/194400/235011965-1914ad72-417a-4b66-9b08-2c6795052d70.png">

Ok. I'm back to square 0. 👌 
Now I can attempt to connect the ***`new`*** AP #207, Sec Cams #130 and Doorbell #201 ... 🤞

**Comment by nelsonic** [comment]
All APs restored and U6-Mesh (outdoor AP #207) working:
https://localhost:8443/manage/9ex79d6p/devices
<img width="1676" alt="home-network-all-aps-restored" src="https://user-images.githubusercontent.com/194400/235014186-47d0c60a-e4d9-4132-aaaa-126a98017706.png">

**Comment by nelsonic** [comment]
MEO Modem / Router only has one Client the UBNT USG (firewall):

<img width="1727" alt="image" src="https://user-images.githubusercontent.com/194400/235014426-fd05e287-caec-4cd0-9a01-bc0cb85c19a8.png">

The router now has a _strong_ password (not default) and is well configured. ✅

**Comment by nelsonic** [comment]
Next is to create a new CAT-6 Cable to connect the 16-port switch to the 8-port PoE switch
because the cable we made a few weeks ago is dead 😢

**Comment by nelsonic** [comment]
Everything:
![IMG_4409](https://user-images.githubusercontent.com/194400/235433461-a1aa43b8-558a-4fb6-a0d5-2aedce886aea.png)
![IMG_4418](https://user-images.githubusercontent.com/194400/235433469-99bf2831-acdf-4b97-aea4-008000dbb3f3.jpeg)

**Comment by nelsonic** [comment]
Wall mounting for 2 cameras and 1 outdoor Access Point _finally_ installed:

![IMG_4436](https://user-images.githubusercontent.com/194400/235435883-a4cfc638-fae6-4a9e-a361-222efe4a7ee3.jpeg)

![IMG_4439](https://user-images.githubusercontent.com/194400/235435888-1c31649e-0917-4e01-999b-5bc53f51ca33.jpeg)

the reason for this design is to protect the gear:

![IMG_4441](https://user-images.githubusercontent.com/194400/235435940-9044d877-410c-40b9-83f5-e1a0504b9272.jpeg)
![IMG_4442](https://user-images.githubusercontent.com/194400/235435944-d9beed62-9000-41e1-a20e-3b794f219420.jpeg)


Cameras still working:

![IMG_4434](https://user-images.githubusercontent.com/194400/235435985-3212734c-a3e5-49b1-a8e8-49b5bfe1ec4d.jpeg)
![IMG_4432](https://user-images.githubusercontent.com/194400/235435987-23e98530-1961-4f5d-a3bb-ced843f19be1.jpeg)

all APs “excellent” signal quality:

![IMG_4435](https://user-images.githubusercontent.com/194400/235436036-f0491a6d-da90-4497-b028-366f76d266e2.png)

# #progress 🚀

**Comment by iteles** [comment]
So cool! I'm of course concerned someone is just going to come along and rip out that expensive camera though, especially with those gates open. 

I'll call magnivertente tomorrow and follow up on the control for opening the gate automagically.

**Comment by nelsonic** [comment]
@iteles indeed. The automatic sliding (locking) gate is the final piece of the puzzle 🧩 but if we could get it moving forward it would be excellent! 👌 ref: https://github.com/dwyl/home/issues/202

as for *this* issue, the next item on my list is the Pi Hole: https://github.com/dwyl/learn-computer-networking/issues/10

**Comment by nelsonic** [comment]
Suuuuuuper annoyingly the Unifi Controller automatically configures all APs to "Meshing" by `default`. 
<img width="1680" alt="unifi-meshing-lame" src="https://user-images.githubusercontent.com/194400/235430482-f9aa3439-feee-4636-b18f-d7150934d8cf.png">

As you can see in this screenshot of the controller, the `DiningRoom-19` 
was meshed and therefore _dependent_ on the `UpstairsCorridor-21` 
which meant that if the "upstream" AP went down the dependent would too.
This is the `default` behaviour for Unifi Networks and it _really_ shouldn't be for APs that are PoE!

So ... I had to reset the whole network just to disable the meshing which wasted another chunk of time!!
Anyway, it's "fixed" now. ✅

**Comment by nelsonic** [comment]
Visiting the Topology tab in the Unifi Console: https://localhost:8443/manage/9ex79d6p/topology

<img width="1155" alt="image" src="https://user-images.githubusercontent.com/194400/236496138-6b11d674-e0aa-4b94-ad93-fa3da1c9298d.png">

We see that there are 3 devices connected to the `USW-Lite-8-PoE`: 
1. `SouthGarden-G3-Flex`
2. `Driveway-G3-Flex`
3. `Outdoor-U6-Mesh`

The `Outdoor-U6-Mesh` in turn has two devices connected:
1. `FrontDoor` - the entrance security camera
2. `G4-Doorbell` #201 

This is exactly as it should be. ✅

**Comment by nelsonic** [comment]
`Pi-hole` working a charm: https://github.com/dwyl/learn-raspberry-pi/issues/26  ✅
